test/Ocelot.UnitTests/Requester/MessageInvokerPoolTests.cs

```diff
@@ -27,12 +28,15 @@ public class MessageInvokerPoolTests : UnitTest
     private HttpContext _context;
     private HttpResponseMessage _response;
     private IWebHost _host;
+    private IOptions<FileConfiguration> _fileConfiguration;
 
     public MessageInvokerPoolTests()
     {
         _ocelotLoggerFactory = new Mock<IOcelotLoggerFactory>();
         _ocelotLogger = new Mock<IOcelotLogger>();
         _ocelotLoggerFactory.Setup(x => x.CreateLogger<MessageInvokerPool>()).Returns(_ocelotLogger.Object);
+        _fileConfiguration = new OptionsWrapper<FileConfiguration>(
+            new FileConfiguration { GlobalConfiguration = new FileGlobalConfiguration() });
```

Comment:
:ok: We will discuss tests later, because the code is not complete.
Not an issue for now!